### DISCUSSION TITLE
Possible DApps to be built using IOTA Smart Contracts

### DISCUSSION THREAD

**Discussion by anistark** [iota-community/X-Team_IOTA_SmartContracts#8] [Ideas]
Let's usual this as a thread to discuss to possible implementation or Decentralised Apps or Smart Contracts to be built using IOTA Smart Contracts.

The goal is to identify only. We can later put all the ideas on wiki so that the community can use that as a base to get started with one or more of those.

**Comment by anistark** [comment]
Escrow

**Comment by anistark** [comment]
Insurance

Claim Settlement, Tracking Beneficiaries etc.

**Comment by anistark** [comment]
Multi-sig Wallet

**Comment by anistark** [comment]
Track Agri-Products

**Comment by anistark** [comment]
Track Auto Parts

**Comment by anistark** [comment]
Land Records.

**Comment by anistark** [comment]
DAO Generator

**Reply by anistark** [reply]
Similar Project: https://aragon.org/

**Comment by anistark** [comment]
Decentralised Energy

**Comment by anistark** [comment]
Decentralised Lending

**Comment by anistark** [comment]
Gambling

**Reply by anistark** [reply]
Similar Project: https://www.bustabit.com/

**Comment by sdellava** [comment]
One of the component of the Distributed IoT platform that I'm working on is the Device Supervisor.
Basically this component takes the device status from a message (or a stream message) and compare it with an expected value. If the data match the supervisor publish a "go" message, otherwise it publish a "no go" message.

To grant that this supervision process can't be tampered with, it need to be executed by a SC.

The supervisor output is a binary value that can be used by any data consumer without need to know out to evaluate the device stats

This SC can be generalized to supervise any kind of expected status and provide an alert in case of mismatch.

**Comment by camusdefinitely** [comment]
Community developed and audited, open-source, Defi 2.0 DAO dapp.

I would love to see our community build a next-generation DeFi D-application that everyone could benefit from. The entire IOTA community benefits as it is built from our own community - not from an eth project ported in that has no real "stake" in the iota community.

I understand it has the potential to get somewhat sticky as all projects involving money do - but if the protocol can be built in a decentralized way, have DAO governance, and an absolutely level playing field - it could be an incredibly powerful addition to the community. Lots of details to iron out and things to think through, but I would love to chat with anyone interested in this idea.
